controllers/front/ajax.php

```diff
@@ -24,13 +24,27 @@ class ps_GoogleanalyticsAjaxModuleFrontController extends ModuleFrontController
 {
     public $ssl = true;
 
+    /**
+     * @var Ps_Googleanalytics
+     */
+    public $module;
```

Comment:
Is this necessary? I think it should be automatic when you are extending ModuleFrontController.

Reply:
No, it's not necessary; I added it to make the type explicit, i.e., "Ps_Googleanalytics", so that there are no warnings in the editor.
However, if you consider it "redundant", I can remove it.